Scripts/RemediationScripts/Remediate-CdnFrontDoorMinTLSVersion.ps1

```diff
@@ -903,109 +701,113 @@ function Reset-FrontDoorRequiredTLSVersion
         Write-Host $([Constants]::SingleDashLine)
     }
     else
     {
-        Write-Host "'Force' flag is provided. Minimum TLS Version will be reset on all Front Doors without any further prompts." -ForegroundColor $([Constants]::MessageType.Warning) -NoNewline
+        Write-Host "'Force' flag is provided. Minimum TLS Version will be reset on all Front Door CDN Profile Domain(s) without any further prompts." -ForegroundColor $([Constants]::MessageType.Warning) -NoNewline
         Write-Host $([Constants]::SingleDashLine)
     }
 
-    Write-Host "[Step 3 of 3] Reset Minimum TLS Version for Front Doors" 
+    Write-Host "[Step 3 of 3] Reset Minimum TLS Version for Front Door CDN Profile Domain(s)" 
     Write-Host $([Constants]::SingleDashLine)
 
-    # Includes Front Doors, to which, previously made changes were successfully rolled back.
-    $frontDoorEndpointsRolledBack = @()
+    # Includes Front Door CDN Profile Domain(s), to which, previously made changes were successfully rolled back.
+    $frontDoorDomainRolledBack = @()
+
+    # Includes Front Door CDN Profile Domain(s) that were skipped during roll back. There were errors rolling back the changes made previously.
+    $frontDoorsDomainsSkipped = @()
 
-    # Includes Front Doors that were skipped during roll back. There were errors rolling back the changes made previously.
-    $frontDoorsEPSkipped = @()
 
-    #$requiredMinTLSVersion = 'TLS10'
 
-    $validFrontDoorEndpointDetails | ForEach-Object {
-        $frontdoorEndpoint = $_
+    $validFrontDoorDomainDetails | ForEach-Object {
+        $frontdoorDomain = $_
         $resourceGroupName = $_.ResourceGroupName
-        $resourceName = $_.ResourceName
+        $resourceName = $_.ProfileName
         $DomainName = $_.DomainName
-        $CertificateType =$_.CertificateType
         $minimumTlsVersion = $_.MinimumTlsVersion
-        $prevMinimumTlsVersion =$_.PrevMinimumTlsVersion
-        $isMinTLSVersionSetOnCustomDomain = $_.isMinTLSVersionSetOnCustomDomain
-        $isMinTLSVersionRolledBackCustomDomainPostRemediation = $false;
-
-        $frontdoorEndpoint | Add-Member -NotePropertyName isMinTLSVersionRolledBackCustomDomainPostRemediation -NotePropertyValue $isMinTLSVersionRolledBackCustomDomainPostRemediation
-        
+        $PrevMinimumTlsVersion = $_.PrevMinimumTlsVersion
         try
         {
-            Write-Host "Fetching Front Door Endpoint configuration: Custom domain Name: [$($DomainName)], Resource Group Name: [$($resourceGroupName)], Resource Name: [$($resourceName)]..." -ForegroundColor $([Constants]::MessageType.Info)
+            Write-Host "Fetching Front Front Door CDN Profiles configuration: Domain Name: [$($DomainName)], Resource Group Name: [$($resourceGroupName)], Resource Name: [$($resourceName)]..." -ForegroundColor $([Constants]::MessageType.Info)
```

Comment:
Statement doesn't make sense here, you are reseting the version so why fetching configurations here,

Reframe the statement, may be in a same way as remediation function